src/StdCheats.sol

```diff
@@ -195,7 +195,7 @@ abstract contract StdCheatsSafe {
     }
 
     // Checks that `addr` is not blacklisted by token contracts that have a blacklist.
-    function assumeNoBlacklisted(address token, address addr) internal virtual {
+    function assumeNoBlacklisted(address token, address addr) internal view virtual {
```

Comment:
Realizing now this method name has a typo, should be `assumeNotBlacklisted`. To avoid a breaking change we should probably just add that version as an alias. Unrelated to this PR so can handle it separately, just noting here

Reply:
Didn't make the change in this PR.

Reply:
Pushed 2feefd2 to make this change